src/dswx_sar/dswx_runconfig.py

```diff
@@ -228,6 +231,7 @@ def validate_group_dict(group_cfg: dict, workflow_name) -> None:
         check_file_path(mgrs_collection_database_file)
         check_file_path(mgrs_database_file)
 
+
 @singledispatch
 def wrap_namespace(ob):
     return ob
```

Comment:
Cannot post the comment directly below, but pylons tells `_wrap_dict()` and `_wrap_list()` is not accessed. Can you check?

Reply:
Yes. `wrap_namespace`  is made to behave differently  depending on the input type, 
 `_wrap_dict()` and `_wrap_list()` are actually being used here. 
https://github.com/opera-adt/DSWX-SAR/blob/cdd2ddddc05db505609fe5298d1a7cc4fd24022b/src/dswx_sar/dswx_runconfig.py#L279